salt/_modules/caasp_etcd.py

```diff
@@ -20,6 +20,10 @@ class NoEtcdServersException(Exception):
     pass
 
 
+def version():
```

Comment:
This method name could be a little confusing.. It's not really the "etcd version", it's the "etcd API version" - maybe rename this to `def api_version()` to avoid confusion?

(And yes, the pillar key suffers from the same issue :()

Reply:
i wonder changing the pillar key would likely force us to do a migration, does it?